.eslintrc.js

```diff
@@ -7,12 +7,16 @@ module.exports = {
     "globals": {
         "Atomics": "readonly",
         "SharedArrayBuffer": "readonly",
-        "define": "readonly"
+        'define': 'readonly',
```

Comment:
I'm a fan of consistency.  What do you think about making these into double quotes?  Not game-breaking.

Reply:
If we can come to consensus, we can add/update editorconfig and eslintrc files to enforce such conventions as only double quotes for string literals in JS, etc.

I think double is preferable to single only in that it is directly paste-portable to JSON, an operation which we may frequently perform on Mongo queries, etc. Beyond that, my preference is that single quotes are more legible.

I'm mostly agnostic/ambivalent when it comes to conventions, but I've discussed some "prevent self-error" type conventions I've adopted before, like always using literals as LRV in equality expressions to prevent accidentally assigning said literal to the variable being compared if I accidentally forget some `=` or similar. I'd be happy to put together a list of this sort of conventions, though much of their enforcement will be left to persistent code hygiene rather than automated tools like static analysis.

Personally, I'm also in favor of us eventually porting all of our code over to TS and leaning heavily on assertions and the like. It would free us up from much manual safety checking we do now, though there are some things they still don't support, like safe navigation operator. This and multiline comment-documented RegExp are some of my most-desired features in TS. Eventual here is a very loose notion, more like "as we find the time".

(As an aside, ESLint can still work with TS, and apparently TSLint will be set aside entirely eventually)